src/back/controllers/api/routes/categories/categories.js

```diff
@@ -4,6 +4,14 @@ const categoryModel = require('./../../models/category');
 const userModel = require('./../../models/user');
 
 const getAll = (req, res, next) => {
+    categoryModel.find({}, (err, categories) => {
+        if (err) next(next);
+
+        res.json(categories);
+    });
+};
+
+const getUserSavedCategories = (req, res, next) => {
     categoryModel.find({}, (err, foundCategories) => {
         if (err) next(err);
```

Comment:
you should add else statement or eturn, to avoid runing any code after next function call
otherwise this will cause error: attempt to send response twice
Generously, I would prefer to use promise instead.